src/Cake.Prca/Orchestrator.cs

```diff
@@ -17,23 +17,23 @@ internal class Orchestrator
         private readonly ICakeLog log;
         private readonly List<ICodeAnalysisProvider> codeAnalysisProviders = new List<ICodeAnalysisProvider>();
         private readonly IPullRequestSystem pullRequestSystem;
-        private readonly ReportIssuesToPullRequestSettings settings;
+        private readonly PrcaSettings settings;
 
         /// <summary>
         /// Initializes a new instance of the <see cref="Orchestrator"/> class.
         /// </summary>
         /// <param name="log">Cake log instance.</param>
         /// <param name="codeAnalysisProviders">List of code analysis issue providers to use.</param>
-        /// <param name="pullRequestSystem">Object for accessing pull request system.</param>
+        /// <param name="pullRequestSystem">Optional object for accessing pull request system.
```

Comment:
Where do we use that? For reading issues we don't need the `Orchestrator` but instead the `IssueReader`.